src/include/distributed/commands/multi_copy.h

```diff
@@ -153,6 +153,14 @@ typedef struct CitusCopyDestReceiver
 	 * when merging into the target tables.
 	 */
 	bool skipCoercions;
+
+	/*
+	 * Skip copying to local placements.
+	 *
+	 * If any of the shards don't have a remote placement, we assume that the
+	 * caller is not doing the right thing and throw an error.
+	 */
+	bool skipCopyToLocalPlacements;
```

Comment:
maybe add a bit more explanation on why we want this, since it seems pretty unusual